src/OrchardCore/OrchardCore.Abstractions/BackgroundJobs/HttpBackgroundJob.cs

```diff
@@ -13,7 +13,7 @@ public static class HttpBackgroundJob
     /// <summary>
     /// Executes a background job in an isolated <see cref="ShellScope"/> after the current HTTP request is completed.
     /// </summary>
-    public static Task ExecuteAfterEndOfRequestAsync(string jobName, Func<ShellScope, Task> job)
+    private static Task ExecuteAfterEndOfRequestInternalAsync(string jobName, Func<ShellScope, Task> job)
```

Comment:
I personally would prefer to move private methods after the public ones. Or in this case leave the existing public method as-is and call that one from the new overloads.

Reply:
I do too. But I think changing it will make it much harder for reviewers to review the come.

I'll move it